phoenix-core/src/it/java/org/apache/hadoop/hbase/regionserver/wal/WALReplayWithIndexWritesAndCompressedWALIT.java

```diff
@@ -176,11 +177,15 @@ private void deleteDir(final Path p) throws IOException {
    */
 @Test
   public void testReplayEditsWrittenViaHRegion() throws Exception {
-    final String tableNameStr = "testReplayEditsWrittenViaHRegion";
-    final RegionInfo hri = RegionInfoBuilder.newBuilder(org.apache.hadoop.hbase.TableName.valueOf(tableNameStr)).setSplit(false).build();
-    final Path basedir = FSUtils.getTableDir(hbaseRootDir, org.apache.hadoop.hbase.TableName.valueOf(tableNameStr));
-    deleteDir(basedir);
-    final TableDescriptor htd = createBasic3FamilyHTD(tableNameStr);
+      final String tableNameStr = "testReplayEditsWrittenViaHRegion";
+      final RegionInfo hri =
+              RegionInfoBuilder.newBuilder(org.apache.hadoop.hbase.TableName.valueOf(tableNameStr))
```

Comment:
Why the fully-qualified TableName?

Reply:
I have no I idea, I've only replaced the class, but didn't touch the arguments.
Will try to pretty this up.